examples/tfengine/modules/team.hcl

```diff
@@ -442,6 +442,36 @@ EOF
   }
 }
 
+# IAM bindings - adding storage admin permissions to a service account.
+template "project_iam" {
```

Comment:
Also add this new dir to https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/tests/tfengine/integration_test.go#L47 so it's tested in the integration tests.